lib/domgen/audit/model.rb

```diff
@@ -47,7 +47,7 @@ def pre_complete
 
             original_entity.jpa.table_name = "vw#{original_entity.name}"
             original_entity.primary_key.sql.generator_type = :sequence
-            original_entity.datetime(:AuditStartAt, :immutable => true, 'jpa.persistent' => false) do |a|
+            end_at_attribtue = original_entity.datetime(:AuditEndAt, :set_once => true, :nullable => true, 'jpa.persistent' => false) do |a|
```

Comment:
This part looks not quite right - The change has removed AuditStartAt and and added duplicate AuditEndAt - so I would assume that this won't work. I assume what you mean to do was assign end at from existing AuditEndAt and keep AuditStartAt